sos/report/plugins/subscription_manager.py

```diff
@@ -85,7 +85,7 @@ def setup(self):
                   "https://subscription.rhsm.redhat.com:443/subscription"
         env = None  # for no_proxy
         try:
-            from rhsm.config import get_config_parser
+            from rhsm.config import get_config_parser  # pylint: disable=C0415
```

Comment:
https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/import-outside-toplevel.html

Can `try` `except` this import at the top-level too. But I think it's fine to keep it here (and disable the warning).